pom.xml

```diff
@@ -109,6 +114,7 @@
 		<object-inspector.version>0.1</object-inspector.version>
 		<scijava-plot.version>0.2.0</scijava-plot.version>
 		<jfreesvg.version>3.2</jfreesvg.version>
+		<scijava-common.version>2.88.0</scijava-common.version>
```

Comment:
I don't understand how the CI build passes. I didn't find version 2.88.0 deployed anywhere... just SNAPSHOT versions. What did I miss?

Reply:
Did you look on the new Sonatype infrastructure? https://s01.oss.sonatype.org/#nexus-search;gav~org.scijava~scijava-common~2.88.0~~

Maybe when you looked it hadn't yet mirrored to Maven Central. Although now it has: https://search.maven.org/search?q=g:org.scijava%20AND%20a:scijava-common%20AND%20v:2.88.0

Reply:
@ctrueden wrote:
> Did you look [on the new Sonatype infrastructure](https://s01.oss.sonatype.org/#nexus-search;gav~org.scijava~scijava-common~2.88.0~~)?

Ah, no, I just checked maven.scijava.org and it wasn't listed anywhere. I'll check directly on s01.oss.sonatype.org as well in the future. Thanks for the hint!

Reply:
> I just checked maven.scijava.org and it wasn't listed anywhere.

Right, maven.scijava.org's public group pulls down artifacts from the various proxies only on demand. So if you look via the web interface and something is missing, you can try e.g. `mvn dependency:get -Dartifact=org.scijava:scijava-common:2.88.0`, and then if you refresh the web view, the artifact will then be there.

I'm not actually sure, but things may also filter into the maven.scijava.org web search every time the indexer runs (which IIRC is set to nightly). But if it's newly deployed that day, then it won't have been indexed that way yet.